content/rc/cloud-integrations/prometheus-integration.md

```diff
@@ -1,18 +1,20 @@
 ---
 Title: Prometheus integration with Redis Cloud
 linkTitle: Prometheus integration
-description: To collect and display metrics data from your databases, you can connect your Prometheus or Grafana server to your Redis Cloud databases.
+description: Use Prometheus and Grafana to collect and visualize Redis Cloud metrics.
 weight: $weight
 alwaysopen: false
 categories: ["RC"]
 ---
 
-To collect and display metrics data from your databases, you can connect your Prometheus or Grafana server to your Redis Cloud subscription.
+You can use Prometheus and Grafana to collect and visualize your Redis Enterprise Cloud metrics.
 
 - [Prometheus](https://prometheus.io/) is an open source systems monitoring and alerting toolkit that can scrape metrics from different sources.
-- [Grafana](https://grafana.com/) is an open source metrics dashboard and graph editor that can process Prometheus data.
+- [Grafana](https://grafana.com/) is an open source metrics visualization tool that can process Prometheus data.
 
-Redis Cloud has an Prometheus compatible endpoint available in order to pull metrics. Prometheus needs to connect to the internal server on port 8070. This is only available on the internal network so [VPC peering](({{< relref "/rc/security/vpc-peering" >}})) is required. VPC peering is only available with Flexible or Annual subscriptions. Because VPC peering is not available on Fixed or Free subscriptions, Prometheus and Grafana cannot connect to databases on Fixed or Free subscriptions.
+Redis Cloud exposes its metrics through a Prometheus endpoint. You can configure your Prometheus server to scrape metrics from your Redis Cloud subscription on port 8070.
+
+The Redis Cloud Prometheus endpoint is exposed on Redis Cloud's internal network. To access this network, you need to enable [VPC peering](({{< relref "/rc/security/vpc-peering" >}})). VPC peering is only available with Flexible or Annual subscriptions. Because VPC peering is not available on Fixed or Free subscriptions, you cannot use Prometheus and Grafana with Fixed or Free subscriptions.
```

Comment:
@cmilesb @banker we now also support node exporter access over Private Service Connect and within a couple of months also over Transit Gateway. I wonder whether we should remove the part about VPC peering and instead just say that this is available for Flexible/Annual subscriptions only.

Reply:
```suggestion
The Redis Cloud Prometheus endpoint is exposed on Redis Cloud's internal network. To access this network, you need to enable [VPC peering]({{< relref "/rc/security/vpc-peering" >}}) or [Private Service Connect]({{< relref "/rc/security/private-service-connect" >}}). Both options are only available with Flexible or Annual subscriptions. You cannot use Prometheus and Grafana with Fixed or Free subscriptions.
```

@joniredis Would this work?